tools/topology/sof-cnl-rt5682.m4

```diff
@@ -0,0 +1,158 @@
+#
+# Topology for Cannonlake with rt5682 codec, Cometlake RVP board.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
+include(`hda.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Apollolake DSP configuration
```

Comment:
@zhuyingjiang typo here? Not Apollolake. 

Also,it is quite confusing to use platform names for the DSP config.It is difficult for someone to tell which one to use unless they know what audio IP's are in all the platforms.
It is clear that multiple platforms use the same DSP config. Can we change to reflect the  audio IP generation instead of platform name? @lgirdwood @plbossart what is your opinion?

Reply:
@ranj063 I will change this PR only for nocodec, the rt5682 related is in #834